api/interfaces.go

```diff
@@ -21,5 +21,5 @@ type MongoServer interface {
 
 // AuthHandler interface for adding auth to endpoints
 type AuthHandler interface {
-	Require(required dpauth.Permissions, handler http.HandlerFunc) http.HandlerFunc
+	Require(required auth.Permissions, handler http.HandlerFunc) http.HandlerFunc
```

Comment:
Based on our earlier conversations I wonder if we should rename the package in this lib to `authZ` to make it clear thats what it is?

Beyond the scope of the PR - just thinking out loud.

Reply:
YES, i vote for such an improvement because it has wasted a whole load of my time.

Reply:
I also agree